app-backend/db/src/main/scala/com/azavea/rf/database/ExportDao.scala

```diff
@@ -116,9 +116,9 @@ object ExportDao extends Dao[Export] {
   }
 
   def getExportStyle(export: Export, exportOptions: ExportOptions, user: User)(implicit xa: Transactor[IO]): ConnectionIO[Either[SimpleInput, ASTInput]] = {
-    (export.projectId, export.toolRunId) match {
-      // Exporting a tool-run
-      case (_, Some(toolRunId)) => astInput(toolRunId, user).map(Right(_))
+    (export.projectId, export.analysisId) match {
+      // Exporting an analysis
+      case (_, Some(analysisId)) => astInput(analysisId, user).map(Right(_))
```

Comment:
Do you want `Some(projectId), Some(analysisId)` to match here -- or should there be error handling for that case?

Reply:
no idea, I didn't dig into that code, just changed the variable name to make sense. I'll investigate later if I have time.